### PR TITLE
7488 e2e tests 5

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7488)

This ticket is for phase 5 of the e2e tests for the 686, mainly to test adding a child AND filling out the 674 together. 

## Testing done
e2e test passes

## Acceptance criteria
- [ ] An e2e test has been created for phase 5 of e2e tests

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 const mock = require('platform/testing/e2e/mock-helpers');
 
+// Helper for filling a date
+const dateFiller = (client, selector, month, day, year) => {
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/tests/e2e/test-helpers.js:4]: Nice 👍

**Review by micahchiang — APPROVED** [review]

**Comment by micahchiang** [comment]
@cohnjesse - Is this pull request ready for review? Looks like it might be.

File: src/platform/testing/e2e/helpers.js [hunk 2]
```diff
@@ -118,6 +118,10 @@ function expectNavigateAwayFrom(client, urlSubstring) {
     .before(Timeouts.slow);
 }
 
+function expectElementNotPresent(client, elementSelector) {
+  client.expect.element(elementSelector).to.not.be.present;
+}
+
```
Comment (bkjohnson) [inline src/platform/testing/e2e/helpers.js:124]: I'm not seeing this used anywhere - is it necessary?

Comment (cohnjesse) [reply src/platform/testing/e2e/helpers.js:124]: I have no idea, I actually didn't add this. no idea how it is in this branch at all.

Comment (rianfowler) [reply src/platform/testing/e2e/helpers.js:124]: have you merged master recently?

Comment (bkjohnson) [reply src/platform/testing/e2e/helpers.js:124]: It appears that you did add it as part of the commit ["added stuff"](https://github.com/department-of-veterans-affairs/vets-website/pull/12995/commits/45780eadfab65faddfaea8fb0edb946e385bcd84)

Comment (cohnjesse) [reply src/platform/testing/e2e/helpers.js:124]: @rianfowler Just merged in

Comment (cohnjesse) [reply src/platform/testing/e2e/helpers.js:124]: @bkjohnson so do I just need to remove it and push?

Comment (bkjohnson) [reply src/platform/testing/e2e/helpers.js:124]: Yes.

Comment (cohnjesse) [reply src/platform/testing/e2e/helpers.js]: @bkjohnson removed and pushed

Comment (bkjohnson) [reply src/platform/testing/e2e/helpers.js]: Cool.  It looks like you can merge if you want since a frontend review isn't required.

**Pull request merged**